### DISCUSSION TITLE
Whether the document should support multiple languages?

### DISCUSSION THREAD

**Discussion by xland** [mikke89/RmlUi#391] [General]
Whether the document should support multiple languages?

**Comment by mikke89** [comment]
I don't understand this question. Take a look at the [localisation documentation](https://mikke89.github.io/RmlUiDoc/pages/localisation.html) for how to support translations.

**Reply by xland** [reply]
The [RmlUi Documentation](https://mikke89.github.io/RmlUiDoc/index.html) is now just  in English.
Whether it can provide versions in other languages?
Or let those who want to translate the documente can submit their translation results.
